### PR TITLE
Cassandra example improvements

### PR DESCRIPTION
**Pull request by JensRantil**

### CODE REVIEW

File: example_configs/cassandra.yml [hunk 1]
```diff
@@ -2,12 +2,19 @@
...
+    destination: "$5"
+    "$1": "$4"
+    "$2": "$3"
+- pattern: org.apache.cassandra.metrics<type=(Connection|Streaming), scope=(\S*), name=(\S*)><>(?:Count|Value)
   name: cassandra_$1_$3
   labels:
     address: "$2"
 - pattern: org.apache.cassandra.metrics<type=(\S*)(?:, ((?!scope)\S*)=(\S*))?(?:, scope=(\S*))?,
-    name=(\S*)><>(Count|Value)
+    name=(\S*)><>(?:Count|Value)
```
Comment (brian-brazil) [inline example_configs/cassandra.yml]: This subgroup is used

File: example_configs/cassandra.yml [hunk 2]
```diff
@@ -2,12 +2,19 @@
 lowercaseOutputLabelNames: true
 lowercaseOutputName: true
 rules:
-- pattern: org.apache.cassandra.metrics<type=(Connection|Streaming), scope=(\S*), name=(\S*)><>(Count|Value)
+- pattern: org.apache.cassandra.metrics<type=(HintedHandOffManager)(?:, ((?!scope)\S*)=(\S*))?(?:, scope=(\S*))?,
```
Comment (brian-brazil) [inline example_configs/cassandra.yml]: Can you give an example of the before and after?

Comment (JensRantil) [reply example_configs/cassandra.yml]: See commit message for a092b3e517f3d655a3e0e07340b65bbf3f2c55b9.

File: example_configs/cassandra.yml (hunk 1 above)
Comment (JensRantil) [reply example_configs/cassandra.yml]: AFAIK, it's not. I can't find a `$5`.

Comment (brian-brazil) [reply example_configs/cassandra.yml]: It's in the name.

File: example_configs/cassandra.yml (hunk 2 above)
Comment (brian-brazil) [reply example_configs/cassandra.yml]: This config doesn't seem to match the example output. Are you sure this is what you're using?

If you could share the help string that'd help my understanding.

File: example_configs/cassandra.yml (hunk 1 above)
Comment (JensRantil) [reply example_configs/cassandra.yml]: Meh, I see now. My apologies.

Comment (JensRantil) [reply example_configs/cassandra.yml]: Force-pushed a fix.

**Comment by brian-brazil** [comment]
This has gone stale, so closing for now.

**Pull request closed without merging**